mappings/net/minecraft/resource/packs/MultiPackResourceManager.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/unmapped/C_xzrqjrss net/minecraft/resource/packs/MultiPackResourceManager
```

Comment:
Quilt Mapping convention: no plural package name.

Also we don't have any dedicated `pack` package, for now put it in `resource`, if we want to subpackage it should be in a separate refactor PR.

Reply:
done